project/src/types/index.ts

```diff
@@ -1,42 +1,70 @@
-import type Comment from './comment';
-import type Offer from './offer';
-import type User from './user';
-import type MapLocation from './map-location';
-import type {City, Cities} from './city';
-import type State from './state';
 import type {
   Actions,
   ThunkActionResult,
   ThunkAppDispatch
 } from './actions';
-import type Token from './token';
-import type AuthData from './auth-data';
-import type AuthInfo from './auth-info';
+
+import type {AuthData} from './auth-data';
+
+import type {
+  RawAuthInfo,
+  AuthInfo
+} from './auth-info';
+
 import type {
-  RawMapLocation,
   RawCity,
+  City,
+  Cities
+} from './city';
+
+import type {CommentPost} from './comment-post';
```

Comment:
это для девочек
```suggestion
export type {CommentPost} from './comment-post';
```
ре-экспорт 👍